cypress/integration/pages/frontPage/tests.js

```diff
@@ -168,42 +194,25 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) =>
             },
           );
         });
-        it('should contain Useful Links if usefulLinks block data exists', () => {
+
+        it('should contain Useful Links if valid usefulLinks block data exists', () => {
           cy.request(`${config[service].pageTypes.frontPage.path}.json`).then(
             ({ body }) => {
               const pageData = body.content.groups;
-              const usefulLinks = pageData.find(data => {
-                return data.type === 'useful-links';
-              });
+              if (shouldDisplayUsefulLinks(pageData)) {
+                // We include the hasStrapline in shouldDisplayUsefulLinks
+                // as we don't render Useful Links if we don't receive a
+                // strapline in the data
```

Comment:
Should this comment be in the `shouldDisplayUsefulLinks` function where we check for validity of the `usefulLinks`?

Reply:
Yes. Moved it to the now  isValidUsefulLinks()